tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 envlist = py38
-skipsdist = true
+skipsdist = True
```

Comment:
these are both treated the same

Reply:
It didn't work for me without this change

Reply:
I believe tox produces an error message for either of them -- but the result is successful (it's because this isn't an actual _library_)